app/pods/components/my-courses/template.hbs

```diff
@@ -1,8 +1,9 @@
 <div class="heading t-align-c mb-5 mt-5">
   My Courses
 </div>
-<div class="row justify-content-center mb-5">
-  {{#if fetchMyCoursesTask.isRunning}}
+<div class="row justify-content-center">
+  {{#if (and (not runs.length) fetchMyCoursesTask.isRunning)}}
+  {{!-- {{#if fetchMyCoursesTask.isRunning}} --}}
     {{loading-card}}
   {{else}}
     {{#if runs.length}}
```

Comment:
You can ditch the `if` block and move the `else` block to an `unless`. 
Also, try not to use more than one conditional helpers, this is an anti-pattern. Use aptly named computed properties instead.